.github/workflows/head-releases.yaml

```diff
@@ -0,0 +1,82 @@
+name : Branch Head Prerelease Images & artifacts (via goreleaser)
+
+on:
+  push:
+    branches:
+      - main
+      - gha-testtesttest
```

Comment:
Do we want to keep the `gha-testtesttest` branch here?

Reply:
No! Great catch - ty.